"blip" login + header & footer changes (rebranding)

jebeck opened the pull request:
I've tentatively scheduled a design review with SK on Wednesday, but I believe this is ready for someone to start looking at in terms of code. It's mostly styling changes with the most substantial component changes being the addition of the `FooterLinks` component separate from the version footer (since there was a request to hide the links in some places but keep the version always present for debugging/support purposes).

Looking for feedback on both how and how much to try to DRY out some of the styles (see TODO comments). Note that since the bulk of this work was done before our meeting re: the style guide project, I have not introduced new styles (e.g., using `flex` to position footer elements) via CSS modules, but I suppose getting a start on that could be one method of DRYing out what's here. Somewhat a timing question. Do we want to wait for the larger new sign-up flow tasks (and larger styles updates per style guide) to start on that?

Will update from master as soon as current v0.17.0 in test is approved for prod.

jebeck commented:
![](https://github.trello.services/images/mini-trello-icon.png) [<ux:1 dev:5 tst:1> [blip] Update Blip header and footer](https://trello.com/c/p5GATqC4/112-ux-1-dev-5-tst-1-blip-update-blip-header-and-footer)

hntrdglss commented on app/components/footerlinks/footerlinks.less:
```diff
@@ -0,0 +1,72 @@
+
+.footer-link {
+  display: inline-block;
+  font-size: 14px;
```
`@font-size-small`?

hntrdglss commented on app/components/footerlinks/footerlinks.less:
```diff
@@ -0,0 +1,72 @@
+@imgHeight: 32px;
+
+.footer-twitter {
+  width: 44.8px;
```
8/10ths of a pixel?! That's very specific. :)

jebeck replied:
Math 🙂  I was scaling exactly to match the size of the Facebook logo from the original dimensions (slightly larger) of the Twitter logo.

hntrdglss commented on app/components/version/version.less:
```diff
@@ -0,0 +1,21 @@
+ */
+
+.Version {
+  font-size: 14px;
```
`@font-size-small`?

hntrdglss commented on app/pages/app/app.less:
```diff
@@ -31,57 +31,27 @@
-      &:first-child {
-        padding-left: 60px;
+      &.footer-section-top {
+        margin-bottom: 10px;
```
`@spacing-small`?

hntrdglss commented on app/pages/emailverification/emailverification.less:
```diff
@@ -26,6 +47,7 @@
 
 .EmailVerification-title {
+  margin-top: @spacing-large;
   margin-bottom: 20px;
```
`@spacing-base`?

hntrdglss commented on app/pages/verificationwithpassword/verificationwithpassword.less:
```diff
@@ -13,15 +13,32 @@
+
 .VerificationWithPassword-form {
-  margin-bottom: @spacing-large;
+  margin-bottom: 5px;
```
`@spacing-tiny`?

hntrdglss commented on app/pages/verificationwithpassword/verificationwithpassword.less:
```diff
@@ -13,15 +13,32 @@
     margin-left: -10px;
     .input-group-explanation {
-      margin: 0px;
+      margin: 40px 0 0 0;
```
`@spacing-large 0 0 0`?

jebeck commented:
Ping @hntrdglss for final review - I believe I've addressed all feedback. Thanks for the catches on swapping in variables, and I factored out a mixin for the flexbox layout on the logged-out routes.

jebeck commented:
Closing with #396 replacing due to tag issue...
